app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/home.scala.html

```diff
@@ -133,14 +133,18 @@ <h2 id="account-details">
     </span>
    </h2>
 
+   <p>
+    @warning(messages("warning.incorrectcalculation"))
+   </p>
```

Comment:
This `<p></p>` tag is not required. Please check existing examples

Reply:
Removed the <p> tag. I wanted to be inline with the existing tags.